packages/shorebird_code_push_protocol/lib/src/models/subscription.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:shorebird_code_push_protocol/shorebird_code_push_protocol.dart';
+
+part 'subscription.g.dart';
+
+/// {@template subscription}
+/// A user's Shorebird subscription.
+/// {@endtemplate}
+@JsonSerializable()
+class Subscription {
+  /// {@macro subscription}
+  Subscription({
+    required this.cost,
+    required this.paidThroughDate,
+    required this.willRenew,
+  });
+
+  /// Creates a [Subscription] from a JSON object.
+  factory Subscription.fromJson(Json json) => _$SubscriptionFromJson(json);
+
+  /// Converts a [Subscription] to a JSON object.
+  Json toJson() => _$SubscriptionToJson(this);
+
+  /// Billing rate, in cents.
```

Comment:
Oh man.  US Cents I assume. :) Someone is going to write an amazing typed "money" object someday. 🤣